### ISSUE TITLE
As a citizen I would like to mark an activity in the current week as completed while offline.

### ISSUE THREAD

**Issue by Nannanas** [aau-giraf/weekplanner-OLDUI#400]
**Is your feature request related to a problem? Please describe.**
This issue is related to the overall implementation of offline functionality.
Link: #9 

Issue #414 must be done first which allows the citizen to be able to view its activities and current weekplan while offline
Selecing activites is done in another issue #777 
Making the acitvities timer work is handling in another issue #402 

Issue aau-giraf/api_client/issues/72 would likely solve this issue

**Describe the solution you'd like**
When a citizen has clicked an activity, it should be possible to mark the activity as completed in offline mode. The mark should be the one chosen in the citizen's settings.
The completed mark should appear both on the activity screen and then weekplan screen as  in online  mode
If the activity has a timer, the timer should be removed from the view as in online mode. If it does not have a timer, it should just be marked as completed like in online mode. 

The cititzen should be able to click on the check mark button to complete an activity
![image](https://user-images.githubusercontent.com/58150918/135491483-9914a232-5f62-41d8-9eb5-2ecc3b00765a.png)

When the activity has been marked as completed and has a timer, it should be removed. If it does not have a timer, the activity should just be marked as completed. 
![image](https://user-images.githubusercontent.com/58150918/135491752-fb50ee5d-773b-47ae-9ca5-8dab6ae25e19.png)

The completed mark should appear on the current weekplan also
![image](https://user-images.githubusercontent.com/58150918/135491895-6d1a2576-3bc7-4acb-8db5-679db30dba37.png)
